lib/contacts.py

```diff
@@ -112,9 +112,9 @@ def resolve_openalias(self, url):
         except DNSException as e:
             print_error('Error resolving openalias: ', str(e))
             return None
-        prefix = 'btc'
+        prefix = 'bch'
```

Comment:
Is that the correct prefix?  We normally use `bitcoincash:` for cashaddr.. is this different somehow? or?  (I know nothing about OpenAlias so just checking).

You sure it's not `bitcoincash:`?

Reply:
I don't think there is anything specified. The other projects using OpenAlias seem to be using ticker symbols. We could use `bitcoincash` of course if we want to, we just need to specify it. I am not aware of any other Bitcoin Cash project using OpenAlias, so it is on us to choose, unless you know of a project that has chosen a prefix already. Note that @janowitz alias uses the `bch` prefix.